GitFyle.Core.Api/Migrations/20240702201800_AddInitialMigration.cs

```diff
@@ -1,4 +1,8 @@
-﻿using System;
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
```

Comment:
Remove the header these are auto generated files by Ef Core so we do not need the header.
Do the same for the rest of the migration files.

Reply:
Removed headers from migration files.